### DISCUSSION TITLE
Easy configuration for bundles extensions

### DISCUSSION THREAD

**Discussion by appelgriebsch** [LazyVim/LazyVim#1193] [Recipes]
Hey,

up to recently I had the extensions coming with LazyVim chosen and strong import in my Nv configuration. Today I found an easier way to let users of my configuration specify which extension they want to use by introducing a custom option object in the `init.lua` (https://github.com/appelgriebsch/Nv/blob/main/init.lua#L8-L27) and later make use of the choices in the `lazy.lua` (https://github.com/appelgriebsch/Nv/blob/main/lua/config/lazy.lua#L37-L72).

Might be something we want to add to the LazyVim starter?!? @folke

**Comment by folke** [comment]
That makes sense! I was actually thinking of creating a small ui where users can simply select the LazyVim extras they want.\
The idea would be that in the ui you can enable/disable a predefined set of specs. For LazyVim extras these would simply be extra imports.

But I'll make it configurable so that users/builders can add other specs to it as well.

I'm leaving on vacation next week, but will look into this when I'm, back.
